lib/src/lds/df1/efdg12.dart

```diff
@@ -1,14 +1,56 @@
 // Created by Crt Vavros, copyright © 2022 ZeroPass. All rights reserved.
 // ignore_for_file: constant_identifier_names
 
+import 'dart:core';
+import 'dart:convert';
 import 'dart:typed_data';
+import 'package:dmrtd/extensions.dart';
+
 import 'dg.dart';
+import '../ef.dart';
+import '../tlv.dart';
 
 class EfDG12 extends DataGroup {
   static const FID = 0x010C;
   static const SFI = 0x0C;
   static const TAG = DgTag(0x6C);
 
+  static const ISSUING_AUTHORITY_TAG = 0x5F19;
+
+  // yyyymmdd
+  static const DATE_OF_ISSUE_TAG = 0x5F26;
+
+  // formatted per ICAO 9303 rules
+  static const NAME_OF_OTHER_PERSON_TAG = 0x5F1A;
+  static const ENDORSEMENTS_AND_OBSERVATIONS_TAG = 0x5F1B;
+  static const TAX_OR_EXIT_REQUIREMENTS_TAG = 0x5F1C;
+
+  // Image per ISO/IEC 10918
+  static const IMAGE_OF_FRONT_TAG = 0x5F1D;
+
+  // Image per ISO/IEC 10918
+  static const IMAGE_OF_REAR_TAG = 0x5F1E;
+
+  // yyyymmddhhmmss
+  static const DATE_AND_TIME_OF_PERSONALIZATION = 0x5F55;
+  static const PERSONALIZATION_SYSTEM_SERIAL_NUMBER_TAG = 0x5F56;
+
+  static const TAG_LIST_TAG = 0x5c;
+
+  DateTime? _dateOfIssue;
+  String? _imageOfFrontTag;
+  String? _imageOfRearTag;
+  String? _issuingAuthority;
+
+  DateTime? get dateOfIssue => _dateOfIssue;
+  String? get imageOfFrontTag => _imageOfFrontTag;
+  String? get imageOfRearTag => _imageOfRearTag;
```

Comment:
In case these 2 properties won't be parsed should be removed including their private members.